content/python/concepts/strings/terms/endswith/endswith.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: '.endswith()'
+Description: 'Checks whether or not a string ends with a given value.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `.endswith()` method checks a value against a given string and returns `True` if the string ends with that value. Otherwise, it returns `False`.
+
+## Syntax
+
+```pseudo
+string.endswith(value, start, end)
+```
+
+Given a `string`, the `.endswith()` method can be used in the following way:
+
+The `value` is a required argument that is checked if it exists at the end of the `string`.  It is also case-sensitive.
+
+Optionally, the `value` can be tested for whether it is at the end of a particular substring within the `string`. This is done within the `start` and `end` index arguments.
```

Comment:
I think these read better as bullet points but I'm wondering about the general opinion on this @Dusch4593 @KTom101 @Pandz18 

If we decide to keep this in paragraph form, can we edit some of the wording?
```suggestion
The required `value` argument is checked if it exists at the end of the `string`.  It is also case-sensitive.

Optionally, the `value` can be tested for whether it is at the end of a particular substring within the `string` using the `start` and `end` index arguments.
```

Reply:
@yangc95 I think it can be condensed a little and changed into bullet points. Something like below maybe? 🤔

```suggestion
-`value` is a required, case-sensitive argument that is checked for at the end of the `string`.

- `value` can also be tested with the optional `start` and `end` index arguments to test whether it is at the end of a particular substring within the `string`.
```

But I think it could also be kept as a paragraph, reworded, and remove line 25 which makes it sound like a list is going to be followed after.

Reply:
@yangc95 I would also think we can change it to a list. I can try to re-word it and will update it as soon as I can!

Reply:
Cool, thanks for the input!

I also think line 25 can be improved upon by mentioning something about the parameters being passed in rather than just "using" the method in certain ways. Alternatively (which I agree with) we can also remove that line as @KTom101 suggested.

If line 25 is removed, I think the rest can stay in paragraph format 👌🏻

Reply:
> Cool, thanks for the input!
> 
> I also think line 25 can be improved upon by mentioning something about the parameters being passed in rather than just "using" the method in certain ways. Alternatively (which I agree with) we can also remove that line as @KTom101 suggested.
> 
> If line 25 is removed, I think the rest can stay in paragraph format 👌🏻

If it's fine with you, I can try to re-word it as a list and if you feel it would be better, leave it as is.